guides/source/action_view_overview.md

```diff
@@ -793,13 +793,13 @@ Form helpers are designed to make working with models much easier compared to us
 
 There are two types of form helpers: those that specifically work with model attributes and those that don't. This helper deals with those that work with model attributes; to see an example of form helpers that don't work with model attributes, check the `ActionView::Helpers::FormTagHelper` documentation.
 
-The core method of this helper, `form_for`, gives you the ability to create a form for a model instance; for example, let's say that you have a model Person and want to create a new instance of it:
+The core method of this helper, `form_with`, gives you the ability to create a form for a model instance; for example, let's say that you have a model Person and want to create a new instance of it:
 
 ```html+erb
-# Note: a @person variable will have been created in the controller (e.g. @person = Person.new)
-<%= form_for @person, url: { action: "create" } do |f| %>
-  <%= f.text_field :first_name %>
-  <%= f.text_field :last_name %>
+<!-- Note: a @person variable will have been created in the controller (e.g. @person = Person.new) -->
+<%= form_with model: @person, url: { action: "create" } do |form| %>
```

Comment:
Let's swap this to `method: :post`.

Reply:
How about I just remove that arg, since it is automatically determined via `model` in the common case.